suite-native/module-accounts/src/navigation/AccountsStackNavigator.tsx

```diff
@@ -28,5 +29,10 @@ export const AccountsStackNavigator = () => (
             name={AccountsStackRoutes.AccountDetail}
             component={AccountDetailScreen}
         />
+        <AccountsStack.Screen
```

Comment:
I am not sure if we should place it under accounts stack since it should be accessible from dashboard in future

Reply:
You can always redirect there from dashboard, if you send the params. But it makes sense to me to add it here, because of the structure right now. If that changes, we can always refactor. But you can even redirect there from settings if you want. It just "belongs" under account detail IMO.

Reply:
I would prefer to add it under RootStackNavigator so it will available from anywhere, should be pretty quick to move it there and it's future proof.

Reply:
fixed